### PR TITLE
feat: `pl-graph` show warnings flag

### PR DESCRIPTION
**Pull request by eliotwrobson**
Adds an attribute to hide warnings raised by `pl-graph` during rendering.

### CODE REVIEW

File: docs/elements.md [hunk 1]
```diff
@@ -1723,6 +1723,7 @@ def generate(data):
...
 | `directed`                  | boolean | true               | Whether to treat edges in an adjacency matrix as directed or undirected. If set to false, then edges will be rendered as undirected. _The input adjacency matrix must be symmetric if this is set to false._                                                            |
 | `weights-presentation-type` | string  | `'f'`              | Number display format for the weights when using an adjacency matrix. If presentation-type is 'sigfig', each number is formatted using the to_precision module to digits significant figures. Otherwise, each number is formatted as `{:.{digits}{presentation-type}}`. |
+| `show-warnings`             | boolean | true               | If set to true, creates issues from warnings raised by Graphviz rendering. Otherwise, hides these warnings.                                                                                                                                                             |
```
Comment (nwalters512) [inline docs/elements.md:1726]: Maybe call this `log-warnings` instead? Let's also not mention issue creation here; the fact that logs create issues is not guaranteed to be true forever.

Comment (eliotwrobson) [reply docs/elements.md:1726]: That makes sense, and I think the name `log-warnings` is clearer.

Comment (eliotwrobson) [reply docs/elements.md]: Done here: https://github.com/PrairieLearn/PrairieLearn/pull/7367/commits/eff1854cbbb900f734e6e7ef9c6d353c03734327

File: docs/elements.md [hunk 2]
```diff
@@ -1723,6 +1723,7 @@ def generate(data):
...
 | `directed`                  | boolean | true               | Whether to treat edges in an adjacency matrix as directed or undirected. If set to false, then edges will be rendered as undirected. _The input adjacency matrix must be symmetric if this is set to false._                                                            |
 | `weights-presentation-type` | string  | `'f'`              | Number display format for the weights when using an adjacency matrix. If presentation-type is 'sigfig', each number is formatted using the to_precision module to digits significant figures. Otherwise, each number is formatted as `{:.{digits}{presentation-type}}`. |
+| `log-warnings`              | boolean | true               | Whether to log warnings that appear as during Graphviz rendering.                                                                                                                                                                                                       |
```
Comment (nwalters512) [inline docs/elements.md:1726]:
```suggestion
| `log-warnings`              | boolean | true               | Whether to log warnings that occur during Graphviz rendering.                                                                                                                                                                                                       |
```

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**